File: app/src/main/java/com/example/myapplication/Fragments/NavigationDrawerBottomSheetFragment.java
package com.example.myapplication.Fragments;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.MenuItem;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import com.example.myapplication.R;
import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
import com.google.android.material.navigation.NavigationView;

public class NavigationDrawerBottomSheetFragment extends BottomSheetDialogFragment {

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {

        View view = inflater.inflate(R.layout.navigation_drawer_bottom_sheet_fragment, null, false);

        NavigationView navigationView = view.findViewById(R.id.nav_view);
        navigationView.setNavigationItemSelectedListener(mOnNavigationItemSelectedListener);

        return view;
    }

    private NavigationView.OnNavigationItemSelectedListener mOnNavigationItemSelectedListener
            = new NavigationView.OnNavigationItemSelectedListener() {

        @Override
        public boolean onNavigationItemSelected(@NonNull MenuItem item) {

            switch (item.getItemId()) {

                case R.id.getting_started:
                    //ContentLoader.loadFragment(new AllStoriesFragment(), getActivity());
                    break;
                case R.id.team:
                    //ContentLoader.loadFragment(new AllStoriesFragment(), getActivity());
                    break;
                case R.id.contact_us:
                    //ContentLoader.loadFragment(new AllStoriesFragment(), getActivity());
                    break;

            }
            return true;
        }
    };
}

File: app/src/main/java/com/havrtz/unfold/fragments/NavigationDrawerBottomSheetFragment.kt
package com.havrtz.unfold.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.havrtz.unfold.R
import com.google.android.material.bottomsheet.BottomSheetDialogFragment
import com.google.android.material.navigation.NavigationView

class NavigationDrawerBottomSheetFragment : BottomSheetDialogFragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.navigation_drawer_bottom_sheet_fragment, null, false)
        val navigationView: NavigationView = view.findViewById(R.id.nav_view)
        navigationView.setNavigationItemSelectedListener(mOnNavigationItemSelectedListener)
        return view
    }

    private val mOnNavigationItemSelectedListener = NavigationView.OnNavigationItemSelectedListener { item ->
        when (item.itemId) {
            R.id.getting_started -> {
            }
            R.id.team -> {
            }
            R.id.contact_us -> {
            }
        }
        true
    }
}